### PR TITLE
added explanation of cosign

### PR DESCRIPTION
**Pull request by Queen-codes**
This PR resolves and closes issue #112 
#### Summary
The overview page for cosign doesn't explain the purpose of cosign and its capabilities. This PR gives a brief explanation of cosign to give the users an understanding of cosign's purpose.

#### Release Note
NONE

#### Documentation
The changes are reflected in the overview page of cosign's section in the docs. Attached below are the changes as reflected on localhost.
Screenshot 2023-04-07 at 15 42 49

### CODE REVIEW

File: content/en/cosign/overview.md [hunk 1]
```diff
@@ -17,25 +17,26 @@ features:
 
 ## Getting Started (Quick Start)
 
-Cosign is part of the Sigstore project. Join us on our [Slack channel](https://sigstore.slack.com/). (Need an [invite](https://links.sigstore.dev/slack-invite)?)
+Cosign is a tool developed as part of the Sigstore project used to verify and sign container images, ensuring their integrity and authenticity.It can also work with blobs, which are binary large objects or files that you can use to store and transfer data.
```
Comment (haydentherapper) [inline content/en/cosign/overview.md:20]: I would shorten this to a single sentence, “Cosign is a command line utility that can sign and verify container images and blobs.”

Comment (ltagliaferri) [reply content/en/cosign/overview.md:20]: What about using the "software artifact, such as container images and blobs" language? Or do we want to move away from that?

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Please take a look at the first sentence again and then we can merge

**Comment by Queen-codes** [comment]
> Please take a look at the first sentence again and then we can merge

alright, thank you

**Comment by Queen-codes** [comment]
Hi @ltagliaferri  and @haydentherapper. Sorry for the late correction. Please, kindly review.

**Review by haydentherapper — APPROVED** [review]

**Review by ltagliaferri — APPROVED** [review]
